README.md

```diff
@@ -66,11 +66,21 @@ docker run --rm -v $(pwd):/workspace ghcr.io/hoverkraft-tech/ci-dokumentor/cli:l
 
 #### GitHub Actions Integration
 
+Using the GitHub Action:
```

Comment:
Add also a link to the packages/docs/content/integrations/github-action.md docs for more information